request_data.py

```diff
@@ -0,0 +1,54 @@
+import emission.core.get_database as edb
+import bson.json_util as bju
+import requests
+import json
+import argparse
+import logging
+
+parser = argparse.ArgumentParser()
+parser.add_argument("from_date",
+        help="from_date in the format of YYYY-MM-DD")
```

Comment:
that does seem like local time.
You can print out the ts and confirm.
At any rate, clarify so that people know what to expect.